evalml/objectives/cost_benefit_matrix.py

```diff
@@ -14,6 +14,8 @@ class CostBenefitMatrix(BinaryClassificationObjective):
     greater_is_better = True
     score_needs_proba = False
     perfect_score = np.inf
+    # Range (-Inf, Inf)
```

Comment:
nit-pick but I thought this was an accidental comment rather than on purpose. Maybe we can move it to be next to `is_bounded_like_percentage` rather than being above?

Reply:
Wouldn't this go next to `perfect_score`?